Assets/Scripts/Inputsensors/MarkerSensor.cs

```diff
@@ -0,0 +1,78 @@
+﻿// <copyright file="MarkerSensor.cs" company="Delft University of Technology">
+// Copyright (c) Delft University of Technology. All rights reserved.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Assets.Scripts.Inputsensors;
+
+/// <summary>
+///  This class keeps track of the location based on Markers.
+/// </summary>
+public class MarkerSensor : ILocationSource, IRotationSource
+{
+    /// <summary>
+    ///   The predicted locations.
+    /// </summary>
+    private List<SensorVector3> locations;
+
+    /// <summary>
+    ///   The predicted rotations.
+    /// </summary>
+    private List<SensorVector3> rotations;
+
+    /// <summary>
+    ///   Gets or sets the MarkerLocations.
+    /// </summary>
+    public MarkerLocations MarkerLocations { get; set; }
+
+    /// <summary>
+    ///   Initializes a new instance of the MarkerSensor class.
+    /// </summary>
+    public MarkerSensor()
+    {
+        this.MarkerLocations = new MarkerLocations("./Assets/Maps/MarkerMap01.xml");
```

Comment:
It is probably better if we store file paths in fields and not in methods/contstructors.